templates/server/puppetserver/conf.d/auth.conf.erb

```diff
@@ -200,16 +242,29 @@ authorization: {
             sort-order: 500
             name: "puppetlabs static file content"
         },
+<%- if scope.function_versioncmp([@server_puppetserver_version, '5.0']) >= 0 -%>
+        {
+            # Allow all users access to the experimental endpoint
+            # which currently only provides a dashboard web ui.
```

Comment:
For the experimental UI? Sure, or just leave it out until the endpoint matures...